src/deepsparse/loggers/helpers.py

```diff
@@ -50,8 +50,52 @@ def possibly_extract_value(value: Any, remainder: Optional[str] = None) -> Any:
     if not remainder:
         return value
 
-    for sub_remainders in remainder.split("."):
-        value = value.__getattribute__(sub_remainders)
+    # splits remainder into separate strings.
+    # each string can access the new nesting depth and optionally
+    # do indexing and slicing at this depth
+    for sub_remainder in remainder.split("."):
```

Comment:
we can't use split here since a `.` since doesn't necessarily imply a property lookup.

for example:
`my_dict["some.key.like.this"].foo`